pkg/debugapi/topology_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package debugapi_test
+
+import (
+	"errors"
+	"net/http"
+	"testing"
+
+	"github.com/ethersphere/bee/pkg/debugapi"
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/jsonhttp/jsonhttptest"
+	topmock "github.com/ethersphere/bee/pkg/topology/mock"
+)
+
+func TestTopologyOK(t *testing.T) {
+	marshalFunc := func() ([]byte, error) {
+		return []byte("abcd"), nil
```

Comment:
well, i am mocking the response of the `MarshalJSON` method on kademlia here, since the point of this test is to test that the debug api works correctly, not that the `MarshalJSON` method (for which there is a separate test in the `kademlia` package) works correctly.

Reply:
cannot argue with that logic